### DISCUSSION TITLE
Help offered

### DISCUSSION THREAD

**Discussion by megawubs** [cdubz/pfnj#51] [Q&A]
Hi 👋 

I would like to help out with this project. Any Idea where I could start?

I've been looking through the issues and found this one: https://github.com/kcal-app/kcal/issues/22. It seems like an easy first issue, any thoughts on it?

**Comment by cdubz** [comment]
👋 Unfortunately this project is _close_ to abandoned but if you've got an interest in the project I'll be happy to review and try to get back in to it. Once I got this off the ground I decided I didn't love Laravel for this sort of app (as compared to Django) so progress has fizzled. However, we still use the app every day personally so I'm hopeful about getting back in to it eventually.

All that said #22 _would_ be a nice place to start. Probably my most annoying missing feature 😄

**Reply by megawubs** [reply]
I'm not new to Laravel (It's been my day job for the last 8 years). So, i'm happy to help!

**Reply by cdubz** [reply]
Nice! Appreciate any thoughts you have on the overall architecture of things, as well. One thing I got bogged down in was feeling like I have way too much logic going on in blade templates. Not sure if that's more common in Laravel or if I just wasn't grasping things well there.

**Reply by megawubs** [reply]
I've opened my first pr in #53. Let me know what you think of it. I've seen what you mean by "doing a lot inside the views". I'll think about some tips. 

Also, would you mind opening this repo up for hacktoberfest? See: https://hacktoberfest.com/participation/#pr-mr-details
